connectors/v2/pakartot.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#player';
+
+Connector.artistSelector = 'div.jp-player-artist > a';
+
+Connector.trackSelector = 'div.jp-player-title > a';
+
+Connector.playButtonSelector = '.jp-play';
+
+Connector.getArtist = function() {
+	var text = $(this.artistSelector).text().trim();
```

Comment:
What do you think of an implementation like this:

https://github.com/david-sabata/web-scrobbler/compare/master...inverse:cleanText?expand=1